src/main/haskell/matching-logic/test/Kore/MatchingLogic/ProofSystem/OnePlusOne/ForallAST.hs

```diff
@@ -4,6 +4,8 @@
 {-# LANGUAGE TemplateHaskell            #-}
 {-# LANGUAGE TypeFamilies               #-}
 {-# LANGUAGE ViewPatterns               #-}
+{-# OPTIONS_GHC -fno-warn-missing-signatures #-}
```

Comment:
Why add `no-warn-missing-signatures` here? It shouldn't be hard to add the inferred signatures for whatever was missing.